templates/foreman.service-overrides.erb

```diff
@@ -2,6 +2,6 @@
 User=<%= scope['foreman::user'] %>
 Environment=FOREMAN_ENV=<%= scope['foreman::rails_env'] %>
 Environment=FOREMAN_HOME=<%= scope['foreman::app_root'] %>
-Environment=FOREMAN_PUMA_THREADS_MIN=<%= scope['foreman::foreman_service_puma_threads_min'] %>
+Environment=FOREMAN_PUMA_THREADS_MIN=<%= @min_puma_threads %>
```

Comment:
it feels a little funny to me that puppet's erb templating supports `@variable` in the local scope from where the template is called. is it more efficient in case the scope object is truly huge? consistently using the same syntax would be nicer to look at otherwise.